src/main/kotlin/build/buf/gradle/BreakingTask.kt

```diff
@@ -14,18 +14,22 @@
 
 package build.buf.gradle
 
+import io.github.g00fy2.versioncompare.Version
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.TaskAction
 
+
 abstract class BreakingTask : DefaultTask() {
     @TaskAction
     fun bufBreaking() {
-        execBuf(
-            "breaking",
-            bufBuildPublicationFile,
-            "--against",
-            singleFileFromConfiguration(BUF_BREAKING_CONFIGURATION_NAME),
-        ) {
+        val args = ArrayList<Any>()
+        args.add("breaking")
+        if (Version(project.getExtension().toolVersion) < Version("1.32.0")) {
+            args.add(bufBuildPublicationFile)
+        }
+        args.add("--against")
+        args.add(singleFileFromConfiguration(BUF_BREAKING_CONFIGURATION_NAME))
+        execBuf(args) {
```

Comment:
`execBuf` takes varargs, so you're going to want to use the splat operator here: `execBuf(*args.toTypedArray)`.

Reply:
As it happens it also takes `Iterable<Any>`

Reply:
Ah, the things you forget when you haven't looked at code in a year.